clang/include/clang/Basic/DiagnosticASTKinds.td

```diff
@@ -998,6 +998,9 @@ def warn_npot_ms_struct : Warning<
   "data types with sizes that aren't a power of two">,
   DefaultError, InGroup<IncompatibleMSStruct>;
 
+def err_itanium_layout_unimplemented : Error<
+  "sorry, Itanium-compatible layout is unimplemented for the current C++ ABI">;
```

Comment:
this isn't really a good error format, please try to make this look more like our other diagnostics.